### DISCUSSION TITLE
Metadata lost when editing a pic

### DISCUSSION THREAD

**Discussion by imsodin** [deckerst/aves#993] [Q&A]
I am traveling, using this awesome app to view and sort my pics - thanks a lot for making it!

As I am on mobile only and not really a mobile native, I don't trust my research of existing issues nor have there patience to write a good issue. I hope I didn't miss an existing discussion and this new one to get pointers is good form :)

When I edit a pic the resulting pic loses metadata, e.g. capture duration or focal length. That happens both on the copy created by snapseed or the replaced file by the default gallery of grapheneos. Given metadata handling seems like a core part of aves, I'd expect it to copy over tags. So I feel like I am missing something. Any pointers/ideas?

**Comment by deckerst** [comment]
Aves does not provide an editor, so when you edit a picture Aves forwards it to an editor app on your device. Whether the metadata is saved depends on that editor app. For example the editor of Google Photos saves the metadata.

**Reply by imsodin** [reply]
Thanks for confirming. Would it be in scope for aves/a request welcome for (probably optional) feature that aves checks for metadata in the edited file and copies them over from the original of missing? Or alternatively a function to "manually copy-paste" metadata from one image to another.

**Reply by deckerst** [reply]
Not sure. I see the value in this, but i want to avoid options when i can. Maybe a prompt following an edit, with a "do not ask again" toggle.

**Reply by imsodin** [reply]
Yeah get the urge to avoid options. As a use am just always scared about these "don't show again" dialogues when I am not 100% sure what it is about - what if I later need it, where will I find it :)

**Reply by deckerst** [reply]
I get the feeling, that's why in Aves all the `do not ask again` prompts can be toggled back from the `Settings` > `Navigation` > `Confirmation dialogs`.
